slices/libsoup2.4-1.yaml

```diff
@@ -0,0 +1,16 @@
+package: libsoup2.4-1
+
+slices:
+  libs:
+    essential:
```

Comment:
I think you're missing a dependency to `glib-networking_libs`

Reply:
also for `libsoup2.4-common` and `libxml2` (you can skip the former though as it doesn't bring any libraries)